File: app/src/main/java/com/base/base_app/data/model/CredentialToken.java
package com.base.base_app.data.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

/**
 * Created by Yamin on 7/18/2018.
 */

public class CredentialToken {
    @SerializedName("access_token")
    @Expose
    private String accessToken;
    @SerializedName("token_type")
    @Expose
    private String tokenType;
    @SerializedName("expires_in")
    @Expose
    private String expiresIn;
    @SerializedName("refresh_token")
    @Expose
    private String refreshToken;

    public String getAccessToken() {
        return accessToken;
    }

    public void setAccessToken(String accessToken) {
        this.accessToken = accessToken;
    }

    public String getTokenType() {
        return tokenType;
    }

    public void setTokenType(String tokenType) {
        this.tokenType = tokenType;
    }

    public String getExpiresIn() {
        return expiresIn;
    }

    public void setExpiresIn(String expiresIn) {
        this.expiresIn = expiresIn;
    }

    public String getRefrshToken() {
        return refreshToken;
    }

    public void setRefrshToken(String refreshToken) {
        this.refreshToken = refreshToken;
    }

    public CredentialToken(String accessToken, String tokenType, String expiresIn, String refreshToken) {
        this.accessToken = accessToken;
        this.tokenType = tokenType;
        this.expiresIn = expiresIn;
        this.refreshToken = refreshToken;
    }
}

File: app/src/main/java/com/kasra/atency/data/model/CredentialToken.kt
package com.kasra.atency.data.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

/**
 * Created by Yamin on 7/18/2018.
 */
data class CredentialToken(
    @field:Expose @field:SerializedName("access_token") var accessToken: String,
    @field:Expose @field:SerializedName(
        "token_type"
    ) var tokenType: String,
    @field:Expose @field:SerializedName("expires_in") var expiresIn: String,
    @field:Expose @field:SerializedName(
        "refresh_token"
    ) var refrshToken: String
)